### PR TITLE
Sphinx Documentation

### PR DESCRIPTION
**Pull request by Ciheim**
Setting up Sphinx

### CODE REVIEW

File: docs/.doctrees/FAQ.doctree
Comment (ceblanton) [inline docs/.doctrees/FAQ.doctree:1]: Is this binary file (and the next few) needed, or is it generated by sphinx as an intermediate file or such?

**Review by ceblanton — CHANGES_REQUESTED** [review]
Looks good, except I don't like the binary files in git repos. Are they really needed?

File: docs/.doctrees/FAQ.doctree
Comment (Ciheim) [reply docs/.doctrees/FAQ.doctree:1]: Auto generated and not needed. Removing!

**Review by ceblanton — APPROVED** [review]
This looks great, thank you!

**Pull request merged**